build.sbt

```diff
@@ -3,9 +3,9 @@ import sbtversionpolicy.withsbtrelease.ReleaseVersion
 
 name:="cross-platform-navigation"
 
-ThisBuild / scalaVersion := "2.13.14"
+ThisBuild / scalaVersion := "3.3.3"
 
-crossScalaVersions := Seq(scalaVersion.value, "2.12.20")
+crossScalaVersions := Seq(scalaVersion.value, "2.13.14", "2.12.20")
```

Comment:
If MAPI is the only consumer, I wonder if we can drop 2.12? Not necessarily something for this PR though.

@guardian/mss-admins any thoughts?

Reply:
Yeah, agree that we can drop 2.12 support.

Reply:
Great! Done with 07a1cb186c5d5b9d8cc8f44436f4e2a3468c7d5e.